common/client-core/src/client/base_client/storage/mod.rs

```diff
@@ -94,7 +94,7 @@ impl MixnetClientStorage for Ephemeral {
 pub struct OnDiskPersistent {
     pub(crate) key_store: OnDiskKeys,
     pub(crate) reply_store: fs_backend::Backend,
-    pub(crate) credential_store: PersistentCredentialStorage,
+    pub credential_store: PersistentCredentialStorage,
```

Comment:
How do I fix this? How can I access `credential_store` field in the sdk without changing this?